docs/tutorials/getting-started/customize-your-deployment.md

```diff
@@ -11,6 +11,7 @@ The next step is to customize your deployment by upgrading specific components o
 This last section of this tutorial is designed to present you with clear options for upgrading your deployment. For specific implementation steps, please check out the linked How-to guides.
 
 Components
```

Comment:
Should "Components" be a header of some sort?

Reply:
oof sorry for missing this